Makefile

```diff
@@ -24,6 +24,7 @@ REPOPATH ?= $(ORG)/minishift
 ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
+MINISHIFT_BINARY ?= $(GOPATH)/src/$(REPOPATH)/out/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)
```

Comment:
This is not a valid dependency. 

    make: *** No rule to make target `/Users/hardy/go/src/github.com/minishift/minishift/out/darwin-amd64/minishift', needed by `integration'.  Stop.

I guess we should use `$(GOPATH)/bin/minishift$(IS_EXE)`